### ISSUE TITLE
check all commits in PR

### ISSUE THREAD

**Issue by Lan-Phan** [dguo/check-author-and-committer-action#14]
Thanks for this great action.
There is 1 use-case that my PR contains multiple commits, is it possible for your action to check all commits instead of top commit?
I see that https://github.com/semcelik/commit-author-action support that, but that action doesn't support to check email of committer.

**Comment by cowwoc** [comment]
@dguo Any plans to add this? It's required for correctness.

**Comment by dguo** [comment]
Yep, this makes sense to me. I might not have time to implement it myself in the next few weeks, but I'm open to a PR in the meantime!
